doc/reference/reference_sql/sql_statements_and_clauses.rst

```diff
@@ -496,54 +496,101 @@ Data types may also appear in :ref:`CAST <sql_function_cast>` functions.
 Table constraint definition
 ++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-Syntax:
+A table :ref:`constraint <index-constraints>` restricts the data you can add to the table.
+If you try to insert invalid data on a column, Tarantool throws an error.
 
-:samp:`[CONSTRAINT {constraint-name}] primary-key-constraint | unique-constraint | check-constraint | foreign-key-constraint`
+A table constraint has the following syntax:
 
-|br|
+.. code-block:: sql
 
-.. image:: constraint.svg
-    :align: left
+    [CONSTRAINT [name]] constraint_expression
+
+    constraint_expression:
+      | PRIMARY KEY (column_name, ...)
+      | UNIQUE (column_name, ...)
+      | CHECK (expression)
+      | FOREIGN KEY (column_name, ...) foreign_key_clause
 
-|br|
 
-Define a constraint, which is a table-element used in a CREATE TABLE statement.
+Define a constraint, which is a table element used in a CREATE TABLE statement.
 
-The constraint-name must be an identifier which is valid according to the rules for identifiers.
-The constraint-name must be unique within the table.
+A constraint name must be an identifier that is valid according to the rules for identifiers.
+A constraint name must be unique within the table.
```

Comment:
This is not exactly so. There cannot be another table foreign key (= tuple foreign key) constraint with the same name, but it is possible to have a CHECK or UNIQUE table constraint, as well as any column constraint (= field constraint) with the same name.

Reply:
Changed like this:
```
A constraint name must be unique within the table for a specific constraint type.
For example, the CHECK and FOREIGN KEY constraints can have the same name.
```